notification/messagestatus.go

```diff
@@ -42,9 +42,12 @@ func (stat *MessageStatus) wrap(ctx context.Context, n *namedSender) *MessageSta
 type MessageState int
 
 const (
-	// MessageStateActive should be specified when a message is still active.
-	// This includes things like remotely queued, ringing, or in-progress calls.
-	MessageStateActive MessageState = iota
+	// MessageStateSending should be specified when a message is sending but has not been sent.
+	// This includes things like ringing, or in-progress calls.
+	MessageStateSending MessageState = iota
+
+	//MessageStatePending includes things like remotely queued.
```

Comment:
This should define pending from GoAlert's perspective, also add a leading space for line comments.

```suggestion
	// MessageStatePending indicates a message waiting to be sent.
```